win32/src/win32crypt/win32cryptmodule.cpp

```diff
@@ -1089,10 +1089,12 @@ static PyObject *PyCryptQueryObject(PyObject *self, PyObject *args, PyObject *kw
     }
 
     BOOL bsuccess;
-    Py_BEGIN_ALLOW_THREADS bsuccess =
+    Py_BEGIN_ALLOW_THREADS;
+    bsuccess =
         CryptQueryObject(objecttype, input, contenttype, formattype, flags, &encoding, &contenttypeout, &formattypeout,
                          &hcertstore, &hcryptmsg, (const void **)&context);
-    Py_END_ALLOW_THREADS if (!bsuccess) return PyWin_SetAPIError("CryptQueryObject");
+    Py_END_ALLOW_THREADS;
+    if (!bsuccess) return PyWin_SetAPIError("CryptQueryObject");
 
     switch (formattypeout) {
```

Comment:
This should probably be `switch (contenttypeout)`.  The issue #1859 bug - or reverse, the fact that it worked at all before with one of the cert files (the binary one), was that by chance CERT_QUERY_FORMAT_BINARY == CERT_QUERY_CONTENT_CERT numerically.  And CERT_QUERY_FORMAT_BASE64_ENCODED == CERT_QUERY_CONTENT_CTL caused the numeric context pointer result value - though the test file seems not to have a ctl.
Both cert test files seem to carry single certs only, one binary format, one base64  - no ctl.

Reply:
ouch - thank you! I really should have noticed that was a base64 file 🤦 It's quite odd that it actually "worked" - I did get a PyCTL_CONTEXT from that certificate, and managed to use it as such. But you are clearly correct, thanks. Sadly I'm back to no coverage for PyCTL_CONTEXT but I think I'll leave the changes around that object in-place.